File: EjerciciosClase/Ejercicio if-else numero mayor o menor o igual a 100. md/Numeromayora100.cpp
#include <iostream>
using namespace std;

int main() {
    int numero;
    cout << "Ingrese un número: ";
    cin >> numero;

    if (numero > 100) {
        cout << "El número es mayor a 100" << endl;
    } else {
        cout << "El número es menor o igual a 100" << endl;
    }

    return 0;
}

File: EjerciciosClase/Ejercicio if-else numero mayor o menor o igual a 100. md/Numeromayora100.java
import java.util.Scanner;

public class Numeromayora100 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.print("Ingrese un número: ");
        int numero = entrada.nextInt();

        if (numero > 100) {
            System.out.println("El número es mayor a 100");
        } else {
            System.out.println("El número es menor o igual a 100");
        }
    }
}
